credential-manager/plutus-scripts/CredentialManager/Debug/Scripts.hs

```diff
@@ -0,0 +1,66 @@
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+{-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:conservative-optimisation #-}
+
+{-# HLINT ignore "Unused LANGUAGE pragma" #-}
+
+module CredentialManager.Debug.Scripts where
+
+import CredentialManager.Scripts.ColdCommittee (coldCommitteeScript)
+import CredentialManager.Scripts.ColdNFT (coldNFTScript)
+import CredentialManager.Scripts.Common
+import CredentialManager.Scripts.HotCommittee (hotCommitteeScript)
+import CredentialManager.Scripts.HotNFT (hotNFTScript)
+import PlutusLedgerApi.V1.Value (AssetClass)
+import PlutusLedgerApi.V3 (
+  ColdCommitteeCredential,
+  HotCommitteeCredential,
+ )
+import PlutusTx (
+  CompiledCode,
+  compile,
+  liftCodeDef,
+  unsafeApplyCode,
+ )
+import PlutusTx.Prelude
+
+coldCommittee
```

Comment:
No we cannot - we need the minting script to be PlutusV2 so that the interim committee members can mint their tokens in babbage on mainnet. In the future this could be the case though...